packages/react-effect/README.md

```diff
@@ -84,15 +84,19 @@ You may optionally pass an options object that contains the following keys (all
   ```tsx
   import {renderToString} from 'react-dom/server';
   import {extract} from '@shopify/react-effect/server';
-  import {createApolloBridge} from '@shopify/react-effect-apollo';
+  import {HtmlContext, HtmlManager} from '@shopify/react-html/server';
```

Comment:
Nice catch!

Reply:
doesn't really matter since the example is to show case `extract`, but nice to not mentioned anything that is depreacted.